### PR TITLE
gazebo classic use legacy packages

### PR DESCRIPTION
**Pull request by muellerbernd**
I modified the PKGBUILD of gazebo classic so that it uses the legacy packages

### CODE REVIEW

File: gazebo/PKGBUILD [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
+pkgrel=3
 pkgdesc="A multi-robot simulator for outdoor environments"
 arch=('i686' 'x86_64')
 url="https://classic.gazebosim.org/"
 license=('Apache')
 depends=('boost' 'curl' 'freeglut' 'freeimage' 'tbb' 'libccd' 'libltdl' 'graphviz'
-         'libtar' 'libxml2' 'ogre-1.9' 'protobuf>=2.3.0' 'sdformat-9' 'ignition-math>=6' 'ignition-transport-8'
-         'ignition-cmake>=2' 'ignition-common-3' 'ignition-fuel_tools-4' 'ignition-msgs-5' 'tinyxml2' 'qwt')
+         'libtar' 'libxml2' 'ogre-1.9' 'protobuf>=2.3.0' 'sdformat-9' 'ignition-math-6' 'ignition-transport-8'
+         'ignition-cmake-2' 'ignition-common-3' 'ignition-fuel_tools-4' 'ignition-msgs-5' 'tinyxml2' 'qwt')
```
Comment (acxz) [inline gazebo/PKGBUILD:20]: Can you replace with an `=`'s?

Also thanks for bumping the pkgrel, but make sure to update the .SRCINFO as well with the `makepkg --printsrcinfo > .SRCINFO` command.

**Review by acxz — CHANGES_REQUESTED** [review]

File: gazebo/PKGBUILD (hunk 1 above)
Comment (acxz) [reply gazebo/PKGBUILD:20]: Thx

**Review by acxz — APPROVED** [review]

**Comment by acxz** [comment]
@alexdewar requesting final merge and push to AUR

**Comment by muellerbernd** [comment]
I had problems with ignition-common-3 since the last update of ffmpeg to version so I made a patch. Now ignition-common-3 builds fine. The patch was inspired from [here](https://github.com/obsproject/obs-studio/pull/8376).

**Review by acxz — CHANGES_REQUESTED** [review]
Can you please separate the changes to `ignition-common-3` as a separate PR? separate PKGBUILD, separate PR.

**Comment by muellerbernd** [comment]
closed this issue and created a new one for gazebo-classic  in #54. For ignition-common-3 I created a new PR in #55

**Pull request closed without merging**